src/components/help/help.test.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import Help from ".";
+import Icon from "../icon";
+
+test("renders tooltip when help button is hovered and removes tooltip when unhovered", async () => {
+  const user = userEvent.setup();
+  render(<Help>foo</Help>);
+
+  await user.hover(screen.getByRole("button", { name: "help" }));
+
+  expect(screen.getByRole("tooltip", { name: "foo" })).toBeVisible();
+
+  await user.unhover(screen.getByRole("button", { name: "help" }));
+
+  expect(screen.queryByRole("tooltip")).not.toBeInTheDocument();
+});
+
+test("renders tooltip when help button is focused and removes tooltip when blurred", async () => {
+  const user = userEvent.setup();
+  render(<Help>foo</Help>);
+
+  await user.click(screen.getByRole("button", { name: "help" }));
+
+  expect(screen.getByRole("tooltip", { name: "foo" })).toBeVisible();
+
+  await user.tab();
+
+  expect(screen.queryByRole("tooltip")).not.toBeInTheDocument();
+});
+
+test("removes tooltip when Escape key is pressed", async () => {
+  const user = userEvent.setup();
+  render(<Help>foo</Help>);
+
+  screen.getByRole("button", { name: "help" }).focus();
+
+  expect(screen.getByRole("tooltip", { name: "foo" })).toBeVisible();
+
+  await user.keyboard("{Escape}");
+
+  expect(screen.queryByRole("tooltip")).not.toBeInTheDocument();
+});
+
+test("does not remove tooltip when Enter key is pressed", async () => {
+  const user = userEvent.setup();
+  render(<Help>foo</Help>);
+
+  screen.getByRole("button", { name: "help" }).focus();
+
+  const tooltip = screen.getByRole("tooltip", { name: "foo" });
+
+  expect(tooltip).toBeVisible();
+
+  await user.keyboard("{Enter}");
+
+  expect(tooltip).toBeVisible();
+});
+
+test("renders help button with the tooltip content as its accessible description", async () => {
+  const user = userEvent.setup();
+  render(<Help>foo</Help>);
+
+  const helpButton = screen.getByRole("button", { name: "help" });
+
+  await user.hover(helpButton);
+
+  expect(helpButton).toHaveAccessibleDescription("foo");
+});
+
+test("renders tooltip content as children passed as a node", async () => {
```

Comment:
**suggestion(non-blocking)**: what do you think of this wording tweak for clarity?

```suggestion
test("composed content is rendered inside the tooltip", async () => {
```